gammapy/scripts/make_bg_cube_models.py

```diff
@@ -311,8 +311,7 @@ def stack_observations(fits_path, outdir, overwrite, method='default'):
         bg_cube_model = make_bg_cube_model(observation_table, fits_path, method)
 
         # save model to file
-        outfile = outdir +\
-                 '/bg_cube_model_group{}'.format(group)
+        outfile = outdir + '/bg_cube_model_group{}'.format(group)
```

Comment:
Oh ... we will support Windows!
And, to avoid the appearance of bias, we shall support Apple watch and MS DOS as well!